### DISCUSSION TITLE
Right mouse click for close window

### DISCUSSION THREAD

**Discussion by iskazain** [voxeltycoon/issues#314] [General] [answered]
Hi there! 

I know that you know OpenTTD :-)
There is very comfortable option of right mouse clicking for close active interface windows.

It wil be very nice to add this option to the game.

**Comment by andrewpey** [comment, marked as answer]
The right mouse button is reserved for various context menus, but you can click the middle mouse button to close windows.

Hope this helps!

**Comment by iskazain** [comment]
Im pretty much happy for so quick answer ))) Tnx for this and advise about middle button.
